test/log_average.test.js

```diff
@@ -0,0 +1,63 @@
+/* eslint no-shadow: 0 */
+
+const test = require("tap").test;
+const ss = require("../");
+
+test("log average", function (t) {
+    t.test("cannot calculate for empty lists", function (t) {
+        t.throws(function () {
+            ss.logAverage([]);
+        });
+        t.end();
+    });
+
+    t.test("cannot calculate for lists with negative numbers", function (t) {
+        t.throws(function () {
+            ss.logAverage([-1]);
+        });
+        t.end();
+    });
+
+    t.test("does not overflow for large products", function (t) {
```

Comment:
We can also remove this and the underflow test as they're a tad on the trivial side for this function (would be more interesting for `geometricMean`)